.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,22 @@
+{
+    "configurations": [
+        {
+            "name": "Mac",
+            "includePath": [
+                "${workspaceFolder}/**",
+                "/opt/homebrew/Cellar/netcdf/4.9.0/include"
+            ],
+            "defines": [
+                
+            ],
+            "macFrameworkPath": [
+                "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks"
+            ],
+            "compilerPath": "/usr/bin/clang",
+            "cStandard": "c17",
+            "cppStandard": "c++17",
+            "intelliSenseMode": "macos-clang-arm64"
+        }
+    ],
+    "version": 4
+}
```

Comment:
Was including this vscode config file intentional? It looks useful but might be specific to @ilaflott's setup.

Reply:
@ceblanton it was intentional.  It is specific to a mac using homebrew.  The configuration name should be more specific.  However, don't worry about it ATT.